### PR TITLE
Generate EE catalog

### PR DESCRIPTION
**Pull request by pierallard**
# Allow the creation of EE catalogs and fix bugs

Some features:
- you can create a EE catalog and import it into CE install, no problem. To generate a EE file, you have to add the 6 EE fixtures to the conf file (see enterprise example)
- you don't need to have EE in your composer to create fixtures and products (only CE)
- it only creates missing files (the _accesses and asset files). For now, it gives access to everyone for everything, but it can be improved in a future PR.
- you now have 2 distinct commands to generate fixtures and generate products.
- fix progress bar
- fix user, user_group and user_role configuration (you can now set your own)
- fix 'product' or 'products' filename and conf
- fix some phpdoc
- fix the failing generated columns in products (with channels and disabled locales or currencies)

### CODE REVIEW

File: Generator/CategoryGenerator.php [hunk 1]
```diff
@@ -32,7 +31,7 @@ class CategoryGenerator
     /**
      * {@inheritdoc}
      */
-    public function generate(array $config, $outputDir)
+    public function generate(array $config, $outputDir,  ProgressHelper $progress)
```
Comment (pierallard) [inline Generator/CategoryGenerator.php]: extra space

File: Generator/FixtureGenerator.php [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    /** @var FamilyGenerator */
+    protected $familyGenerator;
+
+    /** @var CategoryGenerator */
+    protected $categoryGenerator;
+
+    /** @var AttributeGroupGenerator */
+    protected $attrGroupGenerator;
+
+	/** @var AttributeOptionGenerator */
```
Comment (pierallard) [inline Generator/FixtureGenerator.php]: extra spaces

File: Generator/FixtureGenerator.php [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+    protected $attributeOptionGenerator;
+
+	/** @var JobGenerator */
```
Comment (pierallard) [inline Generator/FixtureGenerator.php]: extra space

File: Resources/examples/fixtures_enterprise.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                color: white
+        attribute_groups:
+            count: 10
+        attributes:
+            count: 100
+            identifier_attribute: sku
+            localizable_probability: 10
+            scopable_probability: 5
+            localizable_and_scopable_probability: 2
+            force_attributes:
```
Comment (nidup) [inline Resources/examples/fixtures_enterprise.yml:35]: what mean force attribute?

**Comment by nidup** [comment]
GTM

**Pull request merged**